samples/name_domains_values/Coded_Domain_Reliability.csv

```diff
@@ -1,7 +1,7 @@
 Name,Value
-Completely Reliable,A
-Usually Reliable,B
-Fairly Reliable,C
-Not Usually Reliable,D
-Unreliable,E
-Reliability Cannot Be Judged,F
+"Completely Reliable",A
```

Comment:
Quotes were added because some of the GENC country codes have commas in the actual names.  Thus, for consistency, all of the names in the domains generated from the 2525 amplifier list have quotes around them, to accommodate the country codes and allow for the possibility that in the future there may be additional commas embedded inside some of the  names.